datasets/craigslist_bargains/README.md

```diff
@@ -0,0 +1,263 @@
+
+```
+{
+  'agent_info': {
+    'Bottomline': 
+      [
+        'None', 
+        'None'
+      ],
+    'Role': 
+      [
+        'buyer', 
+        'seller'
+      ],
+    'Target': 
+      [
+        7.0, 
+        10.0
+      ]
+  },
+  'agent_turn': 
+    [
+      0, 
+      1, 
+      ...
+    ],
+  'dialogue_acts': {
+    'intent': 
+      [
+        'init-price',
+        'unknown',
+        ...
+      ],
+    'price': 
+      [
+        5.0, 
+        -1.0, 
+        ...
+        ]
+    },
+  'items': {
+    'Category': 
+      [
+        'phone', 
+        'phone'
+      ],
+    'Description': 
+      [
+        'Charge two devices simultaneously on the go..., 
+        ...
+      ],
+    'Images': 
+      [
+        'phone/6149527852_0.jpg', 
+        'phone/6149527852_0.jpg'
+      ],
+    'Price': 
+      [
+        10.0, 
+        10.0
+      ],
+    'Title': 
+      [
+        'Verizon Car Charger with Dual Output Micro USB and ...', 
+        ...
+      ]
+    },
+  'utterance': 
+    [
+      'Hi, not sure if the charger would work for my car...'
+      'It will work...',
+      ...
+    ]
+}
+
+```
+
+### Data Fields
+
+
+- `agent_info`: Information about each of the agents taking part in the dialogue
+  - `Bottomline`: TBD
+  - `Role`: Whether the agent is buyer or seller
+  - `Target`: Target price that the buyer/seller wants to hit in the negotiation
+- `agent_turn`: Agent taking the current turn in the dialogue (`int` index corresponding to `Role` above)
+- `dialogue_acts`: Rules-based information about the strategy of each agent for each turn
+  - `intent`: The intent of the agent at the particular turn (offer, accept, etc.)
+  - `price`: The current item price associated with the intent and turn in the bargaining process. Default value for missing: (`-1`)
+- `items`: Information about the item the agents are bargaining for. **Note that there is an elembet for each of the fields below for each agent**
+  - `Category`: Category of the item
+  - `Description`: Description(s) of the item
+  - `Images`: (comma delimited) strings of image names of the item
+  - `Price`: Price(s) of the item. Default value for missing: (`-1`)
+  - `Title`: Title(s) of the item
+- `utterance`: Utterance for each turn in the dialogue, corresponding to the agent in `agent_turns`. The utterance may be an empty string (`''`) for some turns if multiple dialogue acts take place after an utterance (e.g. there are often multiple dialogue acts associated with the closing of the bargaining process after all utterances have completed to describe the conclusion of the bargaining).
+
+### Data Splits
+
+This dataset contains three splits, `train`, `validation` and `test`. Note that `test` is not provided with `dialogue_acts` information as described above. To ensure schema consistency across dataset splits, the `dialogue_acts` field in the `test` split is populated with the default values: `{"price": -1.0, "intent": ""}`
```

Comment:
tanks for mentioning how missing values are handled :)

Reply:
No prob!